src/otio_fcp_adapter/fcp_xml.py

```diff
@@ -745,12 +745,17 @@ def track_for_element(self, track_element, track_kind, context):
         md_dict = _xml_tree_to_dict(track_element, timeline_item_tags)
         track_metadata = {META_NAMESPACE: md_dict} if md_dict else None
 
+        track_name = (track_metadata.get("fcp_xml", {}).get("@MZ.TrackName",
+                                                           track_name) if track_metadata else track_name)
```

Comment:
```suggestion
        if track_metadata:
            track_name = track_metadata.get("fcp_xml", {}).get("@MZ.TrackName", track_name)
```
Just a suggestion to avoid initializing `track_name` twice if not necessary and perhaps a bit easier on the eyes.

Reply:
done